core/tests/ts-integration/tests/base-token.test.ts

```diff
@@ -170,6 +171,8 @@ describe('base ERC20 contract checks', () => {
         const withdrawalTx = await withdrawalPromise;
         const l2Receipt = await withdrawalTx.wait();
         await waitForBlockToBeFinalizedOnL1(alice, l2Receipt!.blockNumber);
+        // Sleep to give some time for eth watch to process events, l2 l1 log proof will be available only after it's done.
+        await sleep(3000);
```

Comment:
is it possible to create an API method to track it? `sleep` leads to flackiness of tests